### PR TITLE
[search] Support hyphen in the Wi-Fi category.

### PR DESCRIPTION
**Pull request by mpimenov**
@ygorshenin PTAL

### CODE REVIEW

**Comment by ygorshenin** [comment]
Tests?

File: data/categories.txt [hunk 1]
```diff
@@ -5565,35 +5565,35 @@ sk:Oceán
 sw:Bahari
 
 @wifi
-en:WiFi
-ru:WiFi
+en:WiFi|Wi-Fi
+ru:WiFi|Wi-Fi
 ar:وايفاي
```
Comment (jancborchardt) [inline data/categories.txt]: So will it also work for Arabic, Japanese and Chinese, if this is not fixed? Or is that what is done in the `android/res/values/strings.xml` part? :)

File: indexer/categories_holder.cpp [hunk 2]
```diff
@@ -225,9 +225,11 @@ bool CategoriesHolder::ValidKeyToken(StringT const & s)
     return true;
 
   /// @todo We need to have global stop words array for the most used languages.
-  for (char const * token : {"a", "z", "s", "d", "di", "de", "le", "wi", "fi", "ra", "ao"})
```
Comment (ygorshenin) [inline indexer/categories_holder.cpp]: ROFL

**Comment by ygorshenin** [comment]
LGTM

**Comment by ygorshenin** [comment]
@therearesomewhocallmetim PTAL

File: data/categories.txt (hunk 1 above)
Comment (mpimenov) [inline data/categories.txt]: @jancborchardt I don't think we need to change the synonyms for these languages in this PR. I'm not sure if we have to change them at all.

**Comment by mpimenov** [comment]
PTAL

**Comment by therearesomewhocallmetim** [comment]
LGTM

**Comment by mpimenov** [comment]
PTAL & merge

**Comment by mpimenov** [comment]
PTAL

**Comment by jancborchardt** [comment]
Awesome, thanks a lot! :)

**Pull request merged**